sdk/resourcemanager/azure-resourcemanager-keyvault/src/main/java/com/azure/resourcemanager/keyvault/models/Vault.java

```diff
@@ -62,6 +62,13 @@ public interface Vault
      */
     boolean roleBasedAccessControlEnabled();
 
+    /**
+     * Whether the vault can be accessed from public network.
+     *
+     * @return whether the vault can be accessed from public network.
+     */
+    PublicNetworkAccess publicNetworkAccess();
```

Comment:
I thought about using `boolean publicNetworkAccessEnabled()` (as 1. it is 2 value; 2. other service may not use the enum), but use this to be consistent with `Registry` and `ManagedHsm`.